webpack/envs/commonEnv.js

```diff
@@ -0,0 +1,53 @@
+// @ts-check
+
+import path from "path"
+import TerserPlugin from "terser-webpack-plugin"
+import { basePath, env } from "../utils/env"
+
+export const standardDevtool = env.webpackDevtool || "eval"
```

Comment:
Do we envision there ever being anything other than "standard"? Until that needs to be addressed, could we go with something like
- `webpackEnv.devTool
- `webpackEnv.stats`
- `webpackEnv.minimizer`
- `webpackEnv.resolve`

So as to avoid adding overloaded terms.

Reply:
I went back and forth on this, I really didn't want to overload the base terms. Quite literally the ones you are suggesting be overloaded

I think it would be ok to separate out prod and dev, I don't expect those to stay in sync long term anyway.